src/dawn_native/d3d12/TextureD3D12.cpp

```diff
@@ -107,8 +107,8 @@ namespace dawn_native { namespace d3d12 {
         }
     }
 
-    Texture::Texture(TextureBuilder* builder)
-        : TextureBase(builder), mDevice(ToBackend(builder->GetDevice())) {
+    Texture::Texture(Device* device, const TextureDescriptor* descriptor)
+        : TextureBase(device, descriptor), mDevice(device) {
```

Comment:
TextureD3D12 shouldn't have an mDevice but instead use ToBackend(GetDevice()). This need to be addressed eventually but it isn't necessary for this patch.

Reply:
Done